src/hlds_laser_publisher.cpp

```diff
@@ -152,8 +152,8 @@ int main(int argc, char **argv)
 
   try
   {
-    hls_lfcd_lds::LFCDLaser laser(port, baud_rate, io);   
-    laser_pub = node->create_publisher<sensor_msgs::msg::LaserScan>("scan");
+    hls_lfcd_lds::LFCDLaser laser(port, baud_rate, io);
+    laser_pub = node->create_publisher<sensor_msgs::msg::LaserScan>("scan", rclcpp::QoS(5));
```

Comment:
Would you explain to me why you set History depth is `5`?
ROS2 default is `10` 
https://github.com/ros2/rmw/blob/9f2e181306c66ac0f09674086a108d4657611766/rmw/include/rmw/qos_profiles.h#L54

Reply:
There was no specific reason, I have edited to make it 10.